src/collective/liveblog/browser/header.py

```diff
@@ -1,22 +1,11 @@
 # -*- coding: utf-8 -*-
-from five import grok
-from plone.app.layout.viewlets.interfaces import IAboveContent
-from collective.liveblog.interfaces import ILiveblog
-from collective.liveblog.interfaces import IBrowserLayer
+from plone.app.layout.viewlets.common import ViewletBase
 
-grok.templatedir('templates')
 
-
-class Header(grok.Viewlet):
+class Header(ViewletBase):
 
     """A viewlet to include a header in the Liveblog."""
 
-    grok.name('collective.liveblog.header')
-    grok.context(ILiveblog)
-    grok.layer(IBrowserLayer)
-    grok.require('zope2.View')
-    grok.viewletmanager(IAboveContent)
-
     def available(self):
         """Return True if an image has been defined."""
-        return self.context.image is not None
+        return getattr(self.context, 'image', None) is not None
```

Comment:
this makes no sense; are you registering the viewlet against the right context? you must use `ILiveblog`.